andela/taps-client
# #161115790 Add test for Auth module

#### What does this PR do?
    -  Adds test for Auth module

#### Description of Task to be completed?
    - write tests for auth component
    - write tests for auth container

#### How should this be manually tested?
  - run the `git pull` command
  - run `npm test`
  - the coverage report should show that modules/Auth has 100% coverage

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161115790](https://www.pivotaltracker.com/story/show/161115790)

#### Screenshots or gifs (if appropriate)
N/A

#### Questions:
N/A
